descriptors/ModuleDescriptor-template.json

```diff
@@ -4,7 +4,7 @@
   "provides": [
```

Comment:
[MODORDERS-293](https://issues.folio.org/browse/MODORDERS-293) introduced breaking change to acquisitions unit adding `isDeleted` required property. The `acquisitions-units` interface has to be changed to `2.0`